src/pages/admin/Admin.tsx

```diff
@@ -293,11 +293,12 @@ async function page(session: Session | null) {
   const seasons = await getSeasons();
   const activeSeason = await getActiveSeason();
 
+  if (!activeSeason) {
```

Comment:
This has the unintended side effect of not being able to create new seasons when the current one expires :sweat_smile: 
We shoulld probably do something like `activeSeason ?? allTimeSeason` especially since we just need the season for the latest matches for editing.